fastlane/appstoreres/metadata/source/name.txt

```diff
@@ -0,0 +1 @@
+WordPress – Website Builder
```

Comment:
I called this `name.txt` for consistency with the value `deliver` expects and with the naming standard from App Store Connect:

<img width="621" alt="Screen Shot 2022-03-11 at 12 18 32 pm" src="https://user-images.githubusercontent.com/1218433/157783202-2fd9cd0a-22d0-4a48-8453-8c0529738e75.png">

Reply:
Also notice there's only three characters left. Hopefully our translators will know how to keep this under 30 characters total.

If not, we might have to manually edit the longer ones to go back to "WordPress" only for the time being. It would be nice to automate the process, but I feel the release toolkit would be a better place for that.

Reply:
Suggestion: leave a comment in the #polyglots channel of the WPorg Slack (cf pdjUbD-8v-p2#comment-128) to drive the point home.

That additional note in Slack will only be visible by polyglots from the community helping us translate those (but not to our Translation Vendor for Mag16), but that's better than nothing. And in fact, when it comes to the Mag16 our community is pretty reactive in translating new strings quickly, sometimes before we even have time to send the originals to our vendor, so…

> I just sent out the app strings [to our translation vendor].
> […]
> Languages: [list of 9 locales out of 16]. The other languages already had translations so weren’t sent out.
> 
> — Emily, in pxLjZ-6OF-p2#comment-18800